src/env.cpp

```diff
@@ -1407,6 +1407,13 @@ maybe_t<env_var_t> env_stack_t::get(const wcstring &key, env_mode_flags_t mode)
             wcstring_list_t result;
             if (history) history->get_history(result);
             return env_var_t(L"history", result);
+        } else if (key == L"pipestatus") {
+            auto&& js = proc_get_last_job_statuses();
+            wcstring_list_t result;
+            result.reserve(js->size());
+            for (auto&& i : *js)
+                result.emplace_back(to_string(i));
```

Comment:
Either put this on the same line or use braces.

Reply:
ok will do